.gitmodules

```diff
@@ -2,5 +2,5 @@
 	path = testdata/solidity
 	url = https://github.com/ethereum/solidity
 [submodule "integration/substrate/tornado/tornado-cli"]
```

Comment:
This line still shows substrate.

Reply:
Hmm, good catch. Apparently `git mv` didn't change this, trying to just change it manually now.